util/perl/TLSProxy/Message.pm

```diff
@@ -92,9 +92,7 @@ use constant {
     EXT_FORCE_LAST => 0xffff
 };
 
-# SignatureScheme of TLS 1.3, from
-# https://tools.ietf.org/html/draft-ietf-tls-tls13-20#appendix-B.3.1.3
-# TODO(TLS1.3) update link to IANA registry after publication
+# SignatureScheme of TLS 1.3 from RFC8446
```

Comment:
The IANA registry link has been live for a while at https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-signaturescheme if we did want to link to IANA instead of just citing the RFC.

Reply:
Fixed.